api_definition/coepi_api_0.3.0.yml

```diff
@@ -43,6 +43,10 @@ paths:
               - report
```

Comment:
If 0.4.0 is superseding 0.3.0 then to avoid confusion I'd just delete this version - it will be there in Git history if anyone cares.

Reply:
I think both need to be there until we decide to deprecate v3. This is so that app developers can easily find the definition and documentation for both versions. Both versions are named appropriately.